multi-node/aws/pkg/cluster/cluster.go

```diff
@@ -147,12 +147,22 @@ func (c *Cluster) Create(stackBody string) error {
 		return err
 	}
 
+	tags := make([]*cloudformation.Tag, len(c.StackTags))
+	i := 0
+	for k, v := range c.StackTags {
+		key := k
+		value := v
```

Comment:
exactly, unfortunately theres no way to instantiate a string inline as there is no copy constructor that I found. I did change the cloudFormation..Tag{...} assignment to use append rather then the assignment operator.